checkout-form-manager.js

```diff
@@ -520,6 +517,18 @@ function submitCheckout(e) {
     const formData = new FormData(form);
     formData.append('hash', (new URL(document.location)).searchParams.get('hash'));
     formData.append('offer_id', checkoutElement.getAttribute('offerId'));
+    
+    //Adding utm parameters
+    let _urlParams = new URLSearchParams(window.location.search);
+    formData.append('utm_source', _urlParams.get('utm_source')??'');
+    formData.append('utm_campaign', _urlParams.get('utm_campaign')??'');
+    formData.append('utm_content', _urlParams.get('utm_content')??'');
+    formData.append('utm_term', _urlParams.get('utm_term')??'');
+    //Adding Tag & Tag2
+    formData.append('tag', _urlParams.get('tag')??'');
+    formData.append('tag2', _urlParams.get('tag2')??'');
+    //Adding Order page url
+    formData.append('order_page_url', window.location.protocol + "//" + window.location.host + "/" + window.location.pathname);
```

Comment:
Suggestion to use string interpolation to be more readable
```suggestion
    formData.append('order_page_url', `${window.location.protocol }//${window.location.host} /${window.location.pathname}`);
```

Reply:
I agree